files/fr/learn/css/howto/css_faq/index.md

```diff
@@ -179,7 +179,7 @@ Dans l'exemple précédent, le problème apparaît avec des règles destinées 
 }
 ```
 
-### Utiliser le sélecteur `*`
+### Utiliser le sélecteur \*
```

Comment:
I've asked https://github.com/orgs/mdn/discussions/271 since I'm not sure if this should be changed (looking on the production site, the rendering looks OK to me).

Reply:
Yeah, I prefer the code fences in the titles too. Was asked to remove them in other PRs upstream, but maybe that has changed again

Reply:
Ok I'll merge them as is, there's no point in blocking this. That being written, as I'm still not convinced of this convention, I'm not saying I'll respect it elsewhere.

Thanks once again @nschonni and sorry for the wait.

Reply:
Not a problem, and I hope the come back in the future 😄